5.x/crud-how-to.md

```diff
@@ -615,15 +615,14 @@ Please read the relationship [BelongsToMany](#belongstomany) documentation, ever
 ## Operations
 
 
-<a name="overwrite-a-method-on-the-crud-panel-object"></a>
-### Add an Uneditable Input inside Create or Update Operation
+<a name="add-non-editable-input-inside-create-or-update-operation-stripped-request"></a>
+### Add an Uneditable Input inside Create or Update Operation - Stripped Request
 
 You might want to add a new attribute to the Model that gets saved. Let's say you want to add an `updated_by` indicator to the Update operation, containing the ID of the user currently logged in (`backpack_user()->id`).
 
-**Option 1.** Sure, in your `ProductCrudController::setupUpdateOperation()` can do `CRUD::field('updated_by')->type('hidden')->value(backpack_user()->id);`, but because that hidden field is inside the HTML, it opens up the possibility that a malicious actor will edit the value of the input, in the browser.
-
+Backpack uses the `strippedRequest` configuration to determine the fields that should be saved. **By default it will only save fields that have a corresponding CRUD field defined**
```

Comment:
```suggestion
Backpack uses the `strippedRequest` configuration to determine the fields that should be saved. **By default it will only save inputs that have a corresponding CRUD field defined.**
```